translations/base-de.yaml

```diff
@@ -362,23 +361,23 @@ dialogs:
         missingMods: Fehlende Mods
         newMods: Neu installierte Mods
     resourceLoadFailed:
-        title: Resources failed to load
-        demoLinkText: shapez demo on Steam
-        descWeb: "One ore more resources could not be loaded. Make sure you have a
-            stable internet connection and try again. If this still doesn't
-            work, make sure to also disable any browser extensions (including
-            adblockers).<br><br> As an alternative, you can also play the
-            <demoOnSteamLinkText>. <br><br> Error Message:"
-        descSteamDemo: "One ore more resources could not be loaded. Try restarting the
-            game - If that does not help, try reinstalling and verifying the
-            game files via Steam. <br><br> Error Message:"
+        title: Ressourcen konnten nicht geladen werden
+        demoLinkText: shapez Demoversion auf Steam
+        descWeb: "Eine oder mehrere Ressourcen konnten nicht geladen werden. Stell sicher, dass du eine
+            stabile Internetverbindung haben und versuche es erneut. Wenn dies weiterhin nicht
```

Comment:
hast

Reply:
oh mann, wie konnte mir das nur entgehen. Ich sollte das nicht in der Kaffeepause machen :D